src/main/java/io/phasetwo/keycloak/magic/auth/util/AuthenticatorSharedUtils.java

```diff
@@ -0,0 +1,21 @@
+package io.phasetwo.keycloak.magic.auth.util;
+
+import org.keycloak.authentication.AuthenticationFlowContext;
+import org.keycloak.models.AuthenticatorConfigModel;
+
+import java.util.Map;
+
+public final class AuthenticatorSharedUtils {
```

Comment:
Nit. Prefer not to use `...Utils` for naming. Please call this class `Authenticators` per the guava standard.

Reply:
Changed naming.